src/components/MovieCard/index.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  WrapItem,
+  Link,
+  Image,
+  Heading,
+  Text,
+  List,
+  ListItem,
+} from '@chakra-ui/react';
+import CardImage from '../../assets/card-image.png';
+import StarFull from '../../assets/starFull.png';
```

Comment:
about stars, it should be an icon from [phosphor](https://phosphoricons.com/) and then it's better to use a svg icon instead `.png`

Reply:
I will use it when fetching data dynamically!!